src/styles/main.scss

```diff
@@ -1,3 +1,15 @@
+@import "./utils/variables";
+@import "./blocks/header";
+@import "./blocks/nav";
+@import "./blocks/catalog";
+@import "./blocks/product-card";
+@import "./blocks/stars";
+
+html {
```

Comment:
Consider moving general styling to a separate file to keep your main file clean and clear